get_all_auth_tokens.py

```diff
@@ -123,6 +104,54 @@ def main():
     replace_in_file(args.update, 'APIToken "(.*)"', 'APIToken "%s"' % all_auth_tokens[0][1])
 
 
+def make_request(endpoint_url, access_token=None, json_payload=None):
+    if access_token is not None:
+        headers = {'content-type': 'application/json', 'X-SF-TOKEN': access_token}
+    else:
+        headers = {'content-type': 'application/json'}
+
+    if json_payload is not None:
+        req = urllib2.Request(endpoint_url, json.dumps(json_payload), headers=headers)
+    else:
+        req = urllib2.Request(endpoint_url, headers=headers)
+
+    try:
+        resp = urllib2.urlopen(req)
+        res = resp.read()
+    except urllib2.HTTPError, e:
+        if e.code == 201:
+            resp = e
```

Comment:
that is how the script currently handles a http 201.  But since we aren't creating resources I'll remove and let any error kill process